hathor/transaction/resources/transaction_confirmation.py

```diff
@@ -125,7 +125,7 @@ def render_GET(self, request):
                                 'success': {
                                     'summary': 'Success',
                                     'value': {
-                                        'accumulated_weight': 15.4,
+                                        'accumulated_weight': 43237,
```

Comment:
Same as Marcelo mentioned above, this value is actually a string, right? And `accumulated_weight_raw`?

Reply:
Yes, it's fixed now.